six/six/six.cpp

```diff
@@ -3,8 +3,76 @@
 // This product includes software developed at Datadog
 // (https://www.datadoghq.com/).
 // Copyright 2019 Datadog, Inc.
+
 #include <six.h>
 
+#ifndef _WIN32
+// clang-format off
+// handler stuff
+#include <execinfo.h>
+#include <csignal>
+#include <sys/types.h>
+#include <unistd.h>
+
+// logging to cerr
+#include <iostream>
+// clang-format on
+
+core_trigger_t core_dump = NULL;
+
+static inline void core(int sig)
+{
+    signal(sig, SIG_DFL);
+    kill(getpid(), sig);
+}
+
+#    define STACKTRACE_SIZE 100
```

Comment:
nit: is indentation on purpose?

Reply:
yup, indentation was on purpose force by clang-format. I hate it. Once we have this dialed in, I think I'll try to tweak the clang suggested style a little bit - with y'alls permision.

I'll make the stack depth `500` to have ample space. When the stack is traced it doesn't unwind the go stack, so it should be way more than enough.